lib/modules/datasource/git-refs/readme.md

```diff
@@ -0,0 +1,19 @@
+This datasource returns a ref from a Git repository. The `depName` must be a FQDN. To fetch the latest digest of a ref instead of the named ref, specify the ref as the `currentValue` and match on the `currentDigest`. This datasource can be used in combination with [regex managers](https://docs.renovatebot.com/modules/manager/regex/) to keep dependencies up-to-date which are not specifically supported by Renovate.
+
+**Usage Example**
+
+A real-world example for this specific datasource would be maintaining the HEAD digest of the `master` branch of an unversioned and untagged repo in a config file. This can be achieved by configuring a generic regex manager in `renovate.json` for files named `versions.ini`:
```

Comment:
```suggestion
Say you want to maintain the HEAD digest of the `main` branch of an unversioned and untagged repository in a config file.
You would configure a generic regex manager in `renovate.json` for files named `versions.ini`:
```

Changes:

- Replace `master` with `main`
- Use the word `you` to pull the reader in the text
- Write short sentences and paragraphs